File: GuideObjC/GuideObjC/Views/SeparatorView.m
//
//  SeparatorView.m
//  GuideObjC
//
//  Created by LuzanovRoman on 19.12.16.
//  Copyright © 2016 eigengraph. All rights reserved.
//

#import "SeparatorView.h"

@implementation SeparatorView

- (void)awakeFromNib {
    [super awakeFromNib];
    
    for (NSLayoutConstraint *layoutConstraint in self.constraints) {
        if (layoutConstraint.firstAttribute == NSLayoutAttributeHeight) {
            layoutConstraint.constant = 0.5;
            break;
        }
    }
}
@end

File: GuideSwift/GuideSwift/Views/SeparatorView.swift
//
//  SeparatorView.swift
//  GuideSwift
//
//  Created by LuzanovRoman on 13.12.16.
//  Copyright © 2016 eigengraph. All rights reserved.
//

import UIKit

class SeparatorView: UIView {

    override func awakeFromNib() {
        super.awakeFromNib()

        for layoutConstraint in self.constraints {
            if layoutConstraint.firstAttribute == .height {
                layoutConstraint.constant = 0.5
                break
            }
        }
    }
}
